fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1405,4 +1405,7 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean enable_experimental_mv = false;
+  
+    @ConfField
+    public static boolean enable_dict_optimize_routine_load = false;
```

Comment:
How about the stream load?
The stream load and routine load should have the same configuration.

Reply:
We automatically turn on lowcardinality optimization for streaming load and it will not be eliminated